src/openMVG/exif/sensor_width_database/sensor_width_camera_database.txt

```diff
@@ -656,6 +657,7 @@ Contax TVS Digital;7.11
 Contax N Digital;36
 DJI FC300S;6.16
 DJI FC300X;6.16
+#FC300X;6.16
```

Comment:
We will ignore this file for any PR, since we want one feature or one enhancement per PR.